packages/manager/apps/pci-kubernetes/src/api/hooks/useKubernetes.ts

```diff
@@ -299,7 +300,7 @@ export const useOidcProvider = (projectId: string, kubeId: string) =>
 type KubeConfigProps = {
   projectId: string;
   kubeId: string;
-  onError: (cause: Error) => void;
+  onError: (cause: ResponseAPIError) => void;
```

Comment:
I think it's better to have a generic error and then use isApiError function to check if it's really an api error